### PR TITLE
[PKG-1996] rebuild with icu-73.1

### PR DESCRIPTION
**Pull request by rafaelmartins**
Depends on:
- https://github.com/AnacondaRecipes/icu-feedstock/pull/3
- https://github.com/AnacondaRecipes/libxml2-feedstock/pull/10

### CODE REVIEW

**Review by boldorider4 — APPROVED** [review]
LGTM, remember to remove the abs.yaml

**Review by boldorider4 — APPROVED** [review]
LGTM, remember to remove the abs.yaml

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ source:
   sha256: 35a30d83791329429bc04fe44183ce4ab512fcf6968070a7bfba42fc5a0552a9
 
 build:
-  number: 1
+  number: 2
   skip: True  # [win]
   detect_binary_files_with_prefix: true
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:16]: Could you update `libxml2 2.10` to `libxml2 {{ libxml2 }}` as it's already 2.10 in cbc.yaml

**Review by skupr-anaconda — APPROVED** [review]
LGTM but libxml2

**Comment by skupr-anaconda** [comment]
An error on `osx-arm64`:
`tput: unknown terminal "tmux-256color"`

**Comment by rafaelmartins** [comment]
> An error on `osx-arm64`: `tput: unknown terminal "tmux-256color"`

nice finding, thank you very much!

**Pull request merged**